wiki/Game_modes/en.md

```diff
@@ -0,0 +1,48 @@
+<!-- TODO: 
+- retake images to be ASC-compliant (update with respective game mode stubs)
+- add image for selecting game mode through Mode button -->
+
+# Game modes
+
+In osu!, a **game mode** is one of the four distinct gameplay configurations included in the osu! client (osu! \[game mode\], osu!taiko, osu!catch, and osu!mania). All of which are based on, or inspired by, other existing rhythm video games. 
```

Comment:
```suggestion
In osu!, a **game mode** is one of the four distinct gameplay configurations included in the osu! client - osu! (game mode), osu!taiko, osu!catch, and osu!mania. All of which are based on, or inspired by, other existing rhythm video games. 
```
might be better?

Reply:
brackets within parantheses is correct, and works fine. plus using a single em dash at the end is... debatable

Reply:
what about
```suggestion
In osu! (game), a **game mode** is one of the four distinct gameplay configurations included in the osu! client (osu!, osu!taiko, osu!catch, and osu!mania). All of which are based on, or inspired by, other existing rhythm video games. 
```

Reply:
that works too. but im wondering why we don't want the brackets in the first place

Reply:
@TPGPL is it ok if i go ahead and resolve this conversation? the brackets work fine afaik

Reply:
if you think the current version is correct, then its fine. I guess im just not a fan of square brackets as a part of the text 🤷

Reply:
I think it's clear enough that "osu!" means two different things here, since one is listed as a game mode. I find it a little odd that they sit in parenthesis though, considering they're the whole topic. somthn like this is how I would have written

```suggestion
osu! has four **game modes**: [osu!](osu!), [osu!taiko](osu!taiko), [osu!catch](osu!catch), and [osu!mania](osu!mania). Each has unique gameplay mechanics and was originally inspired by other rhythm games.
```

Reply:
Using a colon is actually not a bad idea. does this work well as a compromise?
```suggestion
In osu!, a **game mode** is one of the four distinct gameplay configurations included in the osu! client: osu! (game mode), osu!taiko, osu!catch, and osu!mania. All of which are based on, or inspired by, other existing rhythm video games. 
```
or maybe this?
```suggestion
In osu!, a **game mode** is one of the four distinct gameplay configurations included in the osu! client: osu!, osu!taiko, osu!catch, and osu!mania. All of which are based on, or inspired by, other existing rhythm video games. 
```

Reply:
i would go with clayton's suggestion and use anchor links to specific headers instead of actual wikilinks

Reply:
@cl8n @TPGPL do the suggestions above work well? if so, which one sound better to you guys?

Reply:
```suggestion
osu! has four **game modes**: [osu!](#-osu), [osu!taiko]#-osutaiko), [osu!catch](#-osucatch), and [osu!mania](#-osumania). Each has unique gameplay mechanics and was originally inspired by other rhythm games.
```
i suppose thats the best option for this line